app/controllers/concerns/blacklight/search_context.rb

```diff
@@ -63,11 +64,9 @@ def find_or_initialize_search_session_from_params params
 
     saved_search = searches_from_history.find { |x| x.query_params == params_copy }
 
-    saved_search ||= begin
-      s = Search.create(:query_params => params_copy)
-      add_to_search_history(s)
-      s
-    end
+    saved_search ||= Search.create(query_params: params_copy).tap do |s|
+                       add_to_search_history(s)
+                     end
```

Comment:
`end` at 69, 21 is not aligned with `saved_search ||= Search.create(query_params: params_copy).tap do |s|` at 67, 4.